web/src/auth/SignupPage.js

```diff
@@ -624,9 +624,43 @@ class SignupPage extends React.Component {
     } else if (signupItem.name === "Agreement") {
       return AgreementModal.renderAgreementFormItem(application, required, tailFormItemLayout, this);
     } else if (signupItem.name.startsWith("Text ")) {
-      return (
-        <div dangerouslySetInnerHTML={{__html: signupItem.label}} />
-      );
+      if (signupItem.type) {
```

Comment:
signupItem.type should take effect under fields like gender , bio , tag that can used to select, if these field not exist in signup item, you can add

Reply:
ok，I understand it